x-pack/plugins/fleet/common/openapi/components/schemas/agent_policy.yaml

```diff
@@ -69,6 +69,10 @@ properties:
     type: object
     description: Override settings that are defined in the agent policy. Input settings cannot be overridden. The override option should be used only in unusual circumstances and not as a routine procedure.
     nullable: true
+  advanced_settings:
```

Comment:
We decided to go with a generic `advanced_settings` in openapi for now. We will look at generating the openapi spec from the settings config later.